spec/support/jasmine.json

```diff
@@ -0,0 +1,11 @@
+{
+  "spec_dir": "src",
+  "spec_files": [
+    "**/*[tT]est.js"
```

Comment:
I'd have gone with default `[sS]pec` instead of `[tT]est`.

Reply:
Poh tay to | Po taa to